src/v1/temporal-types.js

```diff
@@ -168,6 +203,21 @@ export class Date {
     Object.freeze(this);
   }
 
+  /**
+   * Create a date object from the given standard JavaScript <code>Date</code>.
+   * Hour, minute, second, millisecond and time zone offset components of the given date are ignored.
+   * @param {global.Date} standardDate the standard JavaScript date to convert.
+   * @return {Date} new date.
+   */
+  static fromStandardDate(standardDate) {
+    verifyStandardDateAndNanos(standardDate, null);
+
+    return new Date(
+      standardDate.getFullYear(),
+      standardDate.getMonth(),
```

Comment:
I think there's a problem about conversion of `month` value from/to standard date as javascript `Date` object stores months as `0` based values - with `0` = `January` and `11` = `December`. It seems to me that we do not consider this difference in conversions.

Maybe add value range checks on temporal type constructors?